.pre-commit-config.yaml

```diff
@@ -57,6 +57,15 @@ repos:
         language: system
         pass_filenames: false
 
+-   repo: local
+    hooks:
```

Comment:
fine for now but like we discussed we may decide we want to have it just as a pre-push hook, if so we can specify stages https://pre-commit.com/#config-stages